5.2/5.2/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _5._2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string word = "информатика";
+
+            string word1 = word.Substring(2, 1) + word.Substring(0, 1) + word.Substring(4, 1) + word.Substring(5, 1) + word.Substring(10, 1);
+            Console.WriteLine("Первое слово: " + word1);
+
+            string word2 = word.Substring(9, 1) + word.Substring(3, 1) + word.Substring(4, 1) + word.Substring(5, 1) + word.Substring(10, 1);
```

Comment:
Все работает, но сделано "в лоб". Можно было брать куски исходного слова длиннее, чем из одной буквы. А одну букву можно брать напрямую через ее индекс, не используя метод Substring
```suggestion
            string word1 = "" + word[2] + word[0] + word.Substring(4, 2) + word[10];
            Console.WriteLine("Первое слово: " + word1);

            string word2 = "" + word[9] + word.Substring(3, 4);
```